src/renderers/dom/shared/DOMProperty.js

```diff
@@ -208,22 +212,34 @@ var DOMProperty = {
   getPossibleStandardName: __DEV__ ? {autofocus: 'autoFocus'} : null,
 
   /**
-   * All of the isCustomAttribute() functions that have been injected.
+   * Checks whether a property name is a writeable attribute.
+   * @method
    */
-  _isCustomAttributeFunctions: [],
+  isWriteable: function(name, value) {
```

Comment:
Maybe `shouldSetAttribute`? Seems like it being used for attributes is an important part.